automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/workspaces/methodconfigs/ImportMethodConfigModal.scala

```diff
@@ -6,10 +6,12 @@ import org.broadinstitute.dsde.firecloud.component._
 import org.broadinstitute.dsde.firecloud.page.MethodTable
 import org.openqa.selenium.WebDriver
 
-class ImportMethodConfigModal(implicit webDriver: WebDriver) extends Modal("import-method-configuration-modal") {
+class ImportMethodConfigModal(implicit webDriver: WebDriver) extends OKCancelModal("import-method-configuration-modal") {
```

Comment:
OKCancelModal assumes the existence of OK/cancel buttons. Actually, given the way this testing works, there's no real need for the `Modal` vs `OKCancelModal` distinction, so we should probably fold them together and just call it `Modal`.